{
	"id": "CVE-2022-32151",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2022-32151",
				"description": "The httplib and urllib Python libraries that Splunk shipped with Splunk Enterprise did not validate certificates using the certificate authority (CA) certificate stores by default in Splunk Enterprise versions before 9.0 and Splunk Cloud Platform versions before 8.2.2203. Python 3 client libraries now verify server certificates by default and use the appropriate CA certificate stores for each library. Apps and add-ons that include their own HTTP libraries are not affected. For Splunk Enterprise, update to Splunk Enterprise version 9.0 and Configure TLS host name validation for Splunk-to-Splunk communications (https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation) to enable the remediation.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:N/AC:L/Au:N/C:P/I:P/A:N",
							"base_score": 6.4,
							"nvd_base_severity": "MEDIUM",
							"temporal_score": 6.4,
							"nvd_temporal_severity": "MEDIUM",
							"environmental_score": 6.4,
							"nvd_environmental_severity": "MEDIUM"
						}
					},
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:N",
							"base_score": 9.1,
							"base_severity": "CRITICAL",
							"temporal_score": 9.1,
							"temporal_severity": "CRITICAL",
							"environmental_score": 9.1,
							"environmental_severity": "CRITICAL"
						}
					},
					{
						"type": "cvss_v31",
						"source": "prodsec@splunk.com",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:N",
							"base_score": 7.4,
							"base_severity": "HIGH",
							"temporal_score": 7.4,
							"temporal_severity": "HIGH",
							"environmental_score": 7.4,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-295"
						]
					},
					{
						"source": "prodsec@splunk.com",
						"cwe": [
							"CWE-295"
						]
					}
				],
				"references": [
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2022-32151"
					},
					{
						"source": "prodsec@splunk.com",
						"url": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation"
					},
					{
						"source": "prodsec@splunk.com",
						"url": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/Updates"
					},
					{
						"source": "prodsec@splunk.com",
						"url": "https://research.splunk.com/application/splunk_protocol_impersonation_weak_encryption_simplerequest/"
					},
					{
						"source": "prodsec@splunk.com",
						"url": "https://www.splunk.com/en_us/product-security/announcements/svd-2022-0601.html"
					}
				],
				"published": "2022-06-15T17:15:08.81Z",
				"modified": "2022-06-24T01:24:08.687Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"vulnerable": true,
														"fix_status": {
															"class": "unknown"
														},
														"package": {
															"cpe": "cpe:2.3:a:splunk:splunk:*:*:*:*:enterprise:*:*:*"
														},
														"affected": {
															"type": "semver",
															"range": [
																{
																	"lt": "9.0"
																}
															]
														}
													}
												]
											},
											{
												"operator": "OR",
												"criterions": [
													{
														"vulnerable": true,
														"fix_status": {
															"class": "unknown"
														},
														"package": {
															"cpe": "cpe:2.3:a:splunk:splunk_cloud_platform:*:*:*:*:*:*:*:*"
														},
														"affected": {
															"type": "semver",
															"range": [
																{
																	"lt": "8.2.2203"
																}
															]
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2022/CVE-2022-32151.json"
		]
	}
}
